unlock_indicator.c

```diff
@@ -261,7 +301,7 @@ xcb_pixmap_t draw_image(uint32_t *resolution) {
     timeinfo = localtime(&rawtime);
 
     if (unlock_indicator &&
-        (unlock_state >= STATE_KEY_PRESSED || auth_state > STATE_AUTH_IDLE || show_clock)) {
```

Comment:
My changes do slightly alter the behavior of the -k option. The indicator is no longer automatically “always on“ when specifying -k. I thought it makes more sense that way since the clock can now exist outside of the indicator.

Iirc this part is related to that change, but I can't look at the code right now, so if you're concerned please test it :)